src/Stratis.Bitcoin.IntegrationTests/MinerTests.cs

```diff
@@ -887,5 +887,85 @@ private void MinerCreateNewBlockValidity()
             //    SetMockTime(0);
             //    mempool.clear();
         }
+
+        [Fact]
+        public void MiningAndPropagatingPOW_MineBlockCheckNodeConsensusTipIsCorrect()
+        {
+            using (NodeBuilder builder = NodeBuilder.Create(this))
+            {
+                CoreNode miner = builder.CreateStratisPowNode(KnownNetworks.RegTest);
+                builder.StartAll();
+                miner.NotInIBD().SetDummyMinerSecret(new BitcoinSecret(new Key(), miner.FullNode.Network));
```

Comment:
Please check other tests how we set up wallet I think @fassadlr did it much more simpler (something like `miner.NotInIBD().WithWallet()`

Reply:
@thefazzer this can be written like this:

```
                CoreNode miner = builder.CreateStratisPowNode(KnownNetworks.RegTest).NotInIBD();
                builder.StartAll();
```

But ensure you have latest from master :)